demo.py

```diff
@@ -23,24 +23,29 @@ def main():
   # Define the supply chain
   #############################################################################
   """
+  # Create keys
+  print "Generate keypair for Alice..."
```

Comment:
@vladimir-v-diaz Actually, the `from __future__...` statement works the other way around:

```
santiago at ~/.../programas/git ✔ python2
Python 2.7.12 (default, Jun 28 2016, 08:31:05) 
[GCC 6.1.1 20160602] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print "kek"
kek
>>> print("kek")
kek
>>> from __future__ import print_function
>>> print("kek")
kek
>>> print "kek"
  File "<stdin>", line 1
    print "kek"
              ^
SyntaxError: invalid syntax
>>> 
```
Regardless of that technicallity, I think it's a good idea to add the include statement so we don't have forward-incompatible code.